core/workspace_svg.js

```diff
@@ -1908,6 +1939,27 @@ Blockly.WorkspaceSvg.prototype.cancelCurrentGesture = function() {
 };
 
 /**
+ * Don't even think about using this function before talking to rachel-fenichel.
+ *
+ * Force a drag to start without clicking and dragging the block itself.  Used
+ * to attach duplicated blocks to the mouse pointer.
+ * @param {!Object} fakeEvent An object with the properties needed to start a
+ *     drag, including clientX and clientY.
+ * @param {!Blockly.BlockSvg} block The block to start dragging.
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.startDragWithFakeEvent = function(fakeEvent,
+    block) {
+  Blockly.Touch.clearTouchIdentifier();
+  // Also sets the touch identifier
+  if (!Blockly.Touch.checkTouchIdentifier(fakeEvent)) {
+    console.log('something went wrong while setting the touch identifier');
```

Comment:
Actually, I'm going to remove this warning entirely.  It's mostly a holdover from dragging code feeling unstable, and I'm less concerned about that right now.